### ISSUE TITLE
Nuitka  >=0.7 , No module named 'tkinter'

### ISSUE THREAD

**Issue by monkeycc** [Nuitka/Nuitka#1454]
Before submitting an Issue, please review the [Issue Guidelines](https://github.com/kayhayen/Nuitka/blob/develop/CONTRIBUTING.md#submitting-an-issue).

* Please check whether the bug was already reported or fixed.

* Please check out if the develop version of Nuitka works better for you.

  Download source, packages [from here](https://nuitka.net/doc/download.html)
  where you will also find instructions how to do it via pip.

If you want to post a problem/bug, to help us understand and resolve your issue
please check that you have provided at least the information below, and discard
up to here:

* Nuitka version, full Python version, flavor, OS, etc. as output by this
  command (it does more than you think, and we are adding more all the time):

  > python -m nuitka --version
```
0.6.19.7
Commercial: None
Python: 3.7.11 (default, Jul 27 2021, 09:42:29) [MSC v.1916 64 bit (AMD64)]
Flavor: Anaconda Python
Executable: E:\anaconda3\envs\python37\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
```

* How did you install Nuitka and Python

  Did you use pip, anaconda, deb, rpm, from source, git clone, then install into
  a virtualenv or not, this is very important usually.

* The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g.
  `numpy`, you are expected to shorten this to the relevant ones.

  > python -m pip freeze
```
astor==0.8.1
atomicwrites==1.4.0
attrs==21.4.0
Babel==2.9.1
bce-python-sdk==0.8.64
certifi==2021.10.8
cfgv==3.3.1
chardet==4.0.0
charset-normalizer==2.0.12
click==8.0.4
colorama==0.4.4
cycler==0.11.0
decorator==5.1.1
distlib==0.3.4
et-xmlfile==1.1.0
filelock==3.6.0
flake8==4.0.1
flake8-import-order==0.18.1
Flask==2.0.3
Flask-Babel==2.0.0
Flask-Cors==3.0.10
fonttools==4.29.1
future==0.18.2
identify==2.4.11
idna==3.3
importlib-metadata==4.2.0
iniconfig==1.1.1
itsdangerous==2.1.0
Jinja2==3.0.3
joblib==1.1.0
kiwisolver==1.3.2
lap==0.4.0
MarkupSafe==2.1.0
matplotlib==3.5.1
mccabe==0.6.1
motmetrics==1.2.0
nodeenv==1.6.0
Nuitka==0.6.19.7
numpy==1.19.3
opencv-python==4.5.5.62
openpyxl==3.0.9
packaging==21.3
paddlepaddle-gpu==2.2.2.post112
paddleslim==2.2.1
paddlex==2.1.0
pandas==1.3.5
Pillow==9.0.1
platformdirs==2.5.1
pluggy==1.0.0
pre-commit==2.17.0
protobuf==3.19.4
py==1.11.0
py-cpuinfo==8.0.0
pycocotools==2.0.4
pycodestyle==2.8.0
pycryptodome==3.14.1
pyflakes==2.4.0
pyparsing==3.0.7
PySimpleGUI==4.48.0
pytest==7.0.1
pytest-benchmark==3.4.1
python-dateutil==2.8.2
pytz==2021.3
PyYAML==6.0
pyzmq==22.3.0
requests==2.27.1
scikit-learn==0.23.2
scipy==1.7.3
Shapely==1.8.1.post1
shellcheck-py==0.8.0.4
six==1.16.0
threadpoolctl==3.1.0
toml==0.10.2
tomli==2.0.1
tqdm==4.63.0
typing_extensions==4.1.1
urllib3==1.26.8
virtualenv==20.13.2
visualdl==2.2.3
Werkzeug==2.0.3
wincertstore==0.2
xmltodict==0.12.0
zipp==3.7.0

```

Problem description：

`pip install Nuitka==0.6.19.7`

The program is normal

But the upgraded version
Nuitka  >=0.7

`PySimpleGUI`

`python -m nuitka    --standalone  --show-progress   --plugin-enable=tk-inter   --nofollow-imports  --output-dir=OUT Dan.py`

Traceback (most recent call last):
  File "H:\hello\Dan.py", line 13, in <module>
  File "H:\hello\PySimpleGUI\__init__.py", line 2, in <module>
    from .PySimpleGUI import *
  File "H:\hello\PySimpleGUI\PySimpleGUI.py", line 116, in <module>
    import tkinter as tk
ModuleNotFoundError: No module named 'tkinter'

**Comment by kayhayen** [comment]
You picked an excellent combination of options there that makes it the expected output. It working before was definitely coincidental. Read warnings of Nuitka, they mean things.
